include/circular_buffer.hpp

```diff
@@ -243,29 +241,29 @@ class circular_buffer
             _is_mmap_allocated = _buffer->_is_mmap_allocated;
             _size = _buffer->_size;
             _data = std::addressof(_buffer->_data);
-            _read_indices = _buffer->_read_indices;
             _claim_strategy = std::addressof(_buffer->_claim_strategy);
 
             return *this;
         }
 
+        [[nodiscard]] constexpr BufferType get_buffer() const noexcept { return circular_buffer(_buffer); };
```

Comment:
We don't usually use `get_` on getters, but I'm fine either way for this function.

Reply:
changed to `buffer()` as suggested.